src/components/Button/Button.js

```diff
@@ -1,6 +1,41 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
+import styled from 'styled-components';
+
+const StyledButton = styled.button.attrs({
+  nature: (props) => {
```

Comment:
The key name `nature` confused me. Does `kind` make any more sense? `type` would make sense, but its already taken.

Reply:
I will try using the name `type` (underneath the `style`)